src/Component/Sender/Sender.php

```diff
@@ -59,6 +67,23 @@ public function send(string $code, array $recipients, array $data = [], array $a
 
         $renderedEmail = $this->rendererAdapter->render($email, $data);
 
+        if (count($arguments) > 5 && $this->senderAdapter instanceof CcAwareAdapterInterface) {
```

Comment:
I'm wondering if we shouldn't cover the case, where someone is giving more than 5 arguments, but the sender is not `CcAwareAdapterInterface`

Reply:
If someone is giving more than 5 arguments and it's not `CCAware...` then it's fulfilling a different contract and should be customized nevertheless... I think so :D